src/content/questionnaire/sections/B.tsx

```diff
@@ -340,7 +322,7 @@ const FormSectionB: FC<FormSectionProps> = ({ SectionOption, refs }: FormSection
           id="section-b-program-abbreviation"
           label="Program Abbreviation"
           name="program[abbreviation]"
-          value={predefinedProgram ? programOption?.abbreviation : program?.abbreviation}
+          value={program?.abbreviation}
```

Comment:
No action needed here – But the abbreviation field is filtering most special characters out from the pre-selected options. Do you recall if that was a requirement from somewhere? I didn't add any character restrictions on the Edit Program > Abbreviation field, I'm guessing I need to revisit that to ensure it's consistent.

Reply:
Yes, this comes from [CRDCDH-35](https://tracker.nci.nih.gov/browse/CRDCDH-35) "Validation: Only alpha-numerical, ‘-‘ and space are allowed. ". Back in the double digit ticket days 😢.

Reply:
Thanks for tracking that down. I'll update the Edit Program page to match this